composer.json

```diff
@@ -21,9 +21,9 @@
   ],
   "require": {
     "ext-json": "*",
-    "php": "^7.0",
```

Comment:
We cannot bump the minimum PHP version yet. We still support it for now

Reply:
Laravel 8 requires PHP version 7.3. Also Laravel 7 already required PHP 7.2.5

I think you should release a new major version.

Reply:
That won't be for the next release. New major is too far off being completed. We are following the old versioning and Laravel version supports until then

Reply:
Then we cannot support Laravel 8, can we? Am I missing something?

Reply:
Yes will just support all since 5.8 till 8